src/routes/(app)/_components/layout/sidebar.svelte

```diff
@@ -2,16 +2,16 @@
   import SidebarItems from './sidebar-items';
   import NavItem from './nav-item.svelte';
   import BackgroundHexagon from './background-hexagon.svelte';
-  import Icon from '$lib/components/icons/icon.svelte';
-  import Icons from '$lib/components/icons/icons';
+  import Icon from '@/lib/components/icons/icon.svelte';
```

Comment:
curious, why @?

Reply:
I really didn't think much of it and just used what the editor recommended me ahah but yeah in this case gonna switch back to `$lib`